AutomatedTesting/Gem/PythonTests/scripting/ScriptCanvasComponent_OnEntityActivatedDeactivated_PrintMessage.py

```diff
@@ -134,55 +130,51 @@ def validate_start_state(entity: EditorEntity, expected_state: str):
             entity.set_start_status(expected_state)
         return True
 
-    def validate_entities_in_level():
-        controller = validate_entity_exist(controller_dict["name"], Tests.controller_exists)
-        state1_correct = validate_start_state(controller, controller_dict["status"])
+    def validate_entities_in_level(self):
+        controller = self.validate_entity_exist(controller_dict["name"], Tests.controller_exists)
+        state1_correct = self.validate_start_state(controller, controller_dict["status"])
 
-        act_tester = validate_entity_exist(activated_dict["name"], Tests.activated_exists)
-        state2_correct = validate_start_state(act_tester, activated_dict["status"])
+        act_tester = self.validate_entity_exist(activated_dict["name"], Tests.activated_exists)
+        state2_correct = self.validate_start_state(act_tester, activated_dict["status"])
 
-        deac_tester = validate_entity_exist(deactivated_dict["name"], Tests.deactivated_exists)
-        state3_correct = validate_start_state(deac_tester, deactivated_dict["status"])
+        deac_tester = self.validate_entity_exist(deactivated_dict["name"], Tests.deactivated_exists)
+        state3_correct = self.validate_start_state(deac_tester, deactivated_dict["status"])
 
         all_states_correct = state1_correct and state2_correct and state3_correct
         Report.critical_result(Tests.start_states_correct, all_states_correct)
```

Comment:
Scripting tools function candidate?

Reply:
migrated this function to the tools file